lte/cloud/go/services/smsd/client_api.go

```diff
@@ -0,0 +1,58 @@
+package smsd
+
+import (
+	"context"
+
+	"magma/lte/cloud/go/services/smsd/protos"
+	"magma/lte/cloud/go/services/smsd/storage"
+	merrors "magma/orc8r/lib/go/errors"
+	"magma/orc8r/lib/go/registry"
+
+	"github.com/golang/glog"
+)
+
+func ListMessages(networkID string) ([]*storage.SMS, error) {
```

Comment:
Let's add an endpoint to get the status of a specific message by PK, rather than only having the option to return all messages.